.github/workflows/publish-package.yaml

```diff
@@ -1,13 +1,11 @@
 name: Publish to npm, Tag and create GH Release
 
 on:
-  push:
-    branches:
-      - main
-
   pull_request:
     branches:
       - main
+  workflow_dispatch:
+    # This allows manual triggering of the workflow
```

Comment:
i think this can stay like this for now. but we should make sure to follow up with the ability to auto publish on new tags if the sdk version is newer.